app/renderer/components/Inspector/Actions.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { Row, Col, Button, Select, Modal, Input, notification, } from 'antd';
+import { actionDefinitions, actionArgTypes } from './shared';
+
+const { STRING, NUMBER } = actionArgTypes;
+
+const Option = { Select };
+
+export default class Actions extends Component {
+
+  startPerformingAction (actionName, action) {
+    const { startEnteringActionArgs, applyClientMethod } = this.props;
+    if (_.isEmpty(action.args)) {
+      applyClientMethod({methodName: action.methodName, args: [], skipScreenshotAndSource: !action.refresh});
+    } else {
+      startEnteringActionArgs(actionName, action);
+    }
+  }
+
+  executeCommand () {
+    const { pendingAction, cancelPendingAction, applyClientMethod, t } = this.props;
+    let {args, action} = pendingAction;
+    let {methodName} = action;
+
+    // Special case for 'startActivity'
+    // TODO: Fix these... args aren't getting through
+    if (action.methodName === 'startActivity') {
+      args = {appPackage: args[0], appActivity: args[1], appWaitPackage: args[2],
```

Comment:
cannot we pass args as a dictionary?

Reply:
For the commands, `args` is an array. I didn't want users to have to input a JSON string so instead I let them put in an array of args that gets converted to JSON and then called on `wd`